modules/addToList.js

```diff
@@ -0,0 +1,9 @@
+import updateBookData from './updateBookData.js';
+import displayLists from './displayLists.js';
+
+export default function addToList(value, completed, index, toDoList) {
+  toDoList.push({ value, completed, index });
```

Comment:
- [ ] ES6 improves JavaScript by introducing new features such as arrow functions, template literals, and classes, which enhance code readability and maintainability. It also provides modules for better organization and code reusability in larger projects, promoting collaboration among developers.
- Kindly refactor your code to use `ES6` 👍